src/common/run_command.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#if defined(HAVE_CONFIG_H)
+#include <config/bitcoin-config.h>
+#endif
+
+#include <common/run_command.h>
```

Comment:
pico-nit: Shouldn't this be in the very first line after the comment?

Reply:
I'll look at the follow up for moving more code, and could address as part of that.